extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -345,6 +350,27 @@ const OHIFCornerstoneViewport = React.memo(props => {
     };
   }, [displaySets, elementRef, viewportId]);
 
+  // Set up the window level action menu in the viewport action corners.
+  useEffect(() => {
+    const wlActionMenu = getWindowLevelActionMenu({
+      viewportId,
+      viewportElem: elementRef.current,
+      displaySets,
+      servicesManager,
+      commandsManager,
+      verticalDirection: AllInOneMenu.VerticalDirection.BottomToTop,
+      horizontalDirection: AllInOneMenu.HorizontalDirection.LeftToRight,
+    });
+
+    viewportActionCornersService.setActionComponent({
+      viewportId,
+      id: 'windowLevelActionMenu',
+      component: wlActionMenu,
+      location: viewportActionCornersService.LOCATIONS.bottomLeft,
+      indexPriority: -100,
+    });
```

Comment:
Let's put this under a configuration, I can imagine someone would don't like this to get added on their viewport